app/src/main/java/com/github/multimatum_team/multimatum/ReminderBroadcastReceiver.kt

```diff
@@ -6,29 +6,40 @@ import android.app.PendingIntent
 import android.content.BroadcastReceiver
 import android.content.Context
 import android.content.Intent
+import android.util.Log
+import android.widget.ListView
+import androidx.activity.viewModels
 import androidx.core.app.NotificationCompat
 import androidx.core.app.NotificationManagerCompat
+import com.github.multimatum_team.multimatum.model.DeadlineAdapter
+import com.github.multimatum_team.multimatum.repository.DeadlineRepository
+import com.github.multimatum_team.multimatum.viewmodel.DeadlineListViewModel
+import kotlinx.coroutines.runBlocking
+import javax.inject.Inject
 
 //this class is a BroadcastReceiver that will send a notification when it is triggered
 class ReminderBroadcastReceiver : BroadcastReceiver() {
+    @Inject
+    lateinit var deadlineRepository: DeadlineRepository
 
-    override fun onReceive(context: Context?, intent: Intent?) {
+    override fun onReceive(context: Context, intent: Intent?) {
         var channelId = "remindersChannel"
```

Comment:
Use `val` by default if you don't need to mutate the variable (same goes for `title`, `content` and `notificationId`).